OJP/OJP_ModesSupport.xsd

```diff
@@ -30,10 +30,14 @@
 	</xs:simpleType>
 	<xs:simpleType name="PrivateModesEnumeration">
 		<xs:annotation>
-			<xs:documentation>[a category of MODE in TMv6] MODEs offered by private individuals </xs:documentation>
+			<xs:documentation>[a category of MODE in TMv6] MODEs offered by private individuals (or companies or closed communities).</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="carPooling"/>
```

Comment:
Any reason you have removed carPooling all together?

Reply:
I think we discussed that vehiclePooling covers all. You disagree?

Reply:
Ok, fine for me then.